### ISSUE TITLE
Can we add support for transformers?

### ISSUE THREAD

**Issue by moneyhotspring** [SysCV/sam-hq#122]

**Comment by stevezkw1998** [comment]
Reconsiderate this proposal
That would definitely make deployment much easier

**Comment by lkeab** [comment]
Yes, can you help in this process? I can also provide the necessary support.

**Comment by IskanderNakipov** [comment]
Hi @lkeab i'd like to elaborate. Will open an issue in transformers repo and start working on soon if you don't mind
